### DISCUSSION TITLE
Property 'name' is missing in type 'Plugin<any>[]' but required in type 'Plugin<any>'

### DISCUSSION THREAD

**Discussion by lixiaofa** [element-plus/element-plus#14604] [Q&A] [answered]
![image](https://github.com/element-plus/element-plus/assets/23230215/ed4def4b-63d0-4ebd-817d-fedeec386902)
@sxzz

**Comment by sxzz** [comment, marked as answer]
Vue Macros 返回的是一个 Plugin 数组。需要修改 `plugins` 的类型为 `PluginOption` （在最新的Rollup中）

**Reply by lixiaofa** [reply]
报错信息可是缺少name： 
![image](https://github.com/element-plus/element-plus/assets/23230215/129b6714-870b-45eb-a725-9bbe4704df37)
应该是Vue Macros 返回的缺少name

且rollup 根本就没有导出 PluginOption 类型
@sxzz

**Reply by lixiaofa** [reply]
具体怎么操作
